wazuh/wazuh-kibana-app
# [FIX] [Styles] Fix filter button styles of Inventory and height `Evolution` card with no data

## Description

This PR fixes:
- Styles of the button to filter in `Modules/Integrity monitoring/Inventory` and `Modules/Vulnerabilities/Inventory` displayed in the details flyout
- The button to filter in `Modules/Integrity monitoring/Inventory` and `Modules/Vulnerabilities/Inventory` displayed in the details flyout is not displayed for any properties when hovering
- Height of `Evolution` card when there is no data for the selected time range
![image](https://user-images.githubusercontent.com/34042064/146345681-da513934-1b03-47b0-ac7c-a0482529014d.png)

## Tests
- Go to the details flyout of:
  -  `Modules/Integrity monitoring/Inventory`
  -  `Modules/Vulnerabilities/Inventory`
 
 For each, check:
  - hover by the value of properties, a button should appear to add a filter to the searching
  - when the filter button is displayed, it should not move the other HTML elements
  - if the property value is so long, the button could be overlaped to the text, the button should be visible without problems. Check for:
    - Light theme
    - Dark theme
  
- Go to `Agents` and check the height of `Evolution` card to match with the height of others cards in the same row when the time range has no data.

Closes #3720
